lldb/include/lldb/Target/RegisterContext.h

```diff
@@ -144,6 +144,8 @@ class RegisterContext : public std::enable_shared_from_this<RegisterContext>,
 
   uint64_t GetPC(uint64_t fail_value = LLDB_INVALID_ADDRESS);
 
+  uint64_t GetThreadPointer(uint64_t fail_value = LLDB_INVALID_ADDRESS);
```

Comment:
Need some header documention here unless we have it elsewhere describing that if there is a regsister that points to thread specific data, like TLS data and other thread specific stuff, this is the register.